extensdotio/sails-hook-responses
# Add keepResponseErrors configuration

* Update to the latest version of Lodash
* Remove deprecated warning for older Sails versions
* Add `keepResponseErrors` config because the behavior in response handlers of clearing the body in production mode should be optional
